### PR TITLE
Fix #566 Updated Proxy section with the relevant steps from VSM.

### LINKED ISSUE #566: Proxy documentation needs to updated with VSM options.
With VSM providing way to set proxy information [2] , ADB documentation [1] needs to be updated with it.
[1] https://github.com/projectatomic/adb-atomic-developer-bundle/blob/master/docs/installing.adoc#set-up-adb-behind-an-http-proxy
[2] https://github.com/projectatomic/vagrant-service-manager#http-proxy-settings

### PR DESCRIPTION
**Pull request by Preeticp**
As part of this issue, reworded the Proxy setup section in vagrant-service-manager readme and added the relevant steps to the proxy section.
Also separated the 'Start ADB' substep from Step 5 and made it into Step 7 as it seemed logically the correct order of steps.

### CODE REVIEW

**Comment by Preeticp** [comment]
@budhrg, @LalatenduMohanty request your review.

File: docs/installing.adoc [hunk 1]
```diff
@@ -293,27 +271,27 @@ $ vagrant box add adb <local path to the downloded image>
...
-export PROXY="<proxy_server>:<port>"
-export PROXY_USER="foo"
-export PROXY_PASSWORD="mysecretpass"
-....
+----
+$ vagrant up
+----
 
-For Windows CMD or Powershell:
+This will download ADB and set it up to work with the provider of
```
Comment (brgnepal) [inline docs/installing.adoc]: Why download ADB again here? It is done in step 5 right?

**Comment by praveenkumar** [comment]
LGTM.

File: docs/installing.adoc [hunk 2]
```diff
@@ -293,27 +271,27 @@ $ vagrant box add adb <local path to the downloded image>
 [[set-up-adb-behind-an-http-proxy]]
 == Step 6. (Optional) Set up ADB behind an HTTP proxy
 
-ADB can be set up behind a proxy server. You need to export the proxy
-server information in to the environment and then run `vagrant up`.
+ADB can be set up behind a proxy server. You will need to set a few proxy parameters in the Vagrantfile you chose in the previous step. Refer the https://github.com/projectatomic/vagrant-service-manager#http-proxy-settings[HTTP Proxy Settings] section of the vagrant-service-manager for further details.
```
Comment (LalatenduMohanty) [inline docs/installing.adoc:304]: Steps to set proxy server information is a very important piece of ADB user experience. So I think this section should contain VSM config options and not redirect users to VSM documentation for the information. It is fine to add the link as a reference @Preeticp @budhrg @praveenkumar comments?

Comment (Preeticp) [reply docs/installing.adoc:304]: Makes sense @LalatenduMohanty!

File: docs/installing.adoc [hunk 3]
```diff
@@ -293,27 +271,37 @@ $ vagrant box add adb <local path to the downloded image>
...
-export PROXY_USER="foo"
-export PROXY_PASSWORD="mysecretpass"
+config.servicemanager.http_proxy = <Proxy URL>
+config.servicemanager.http_proxy_user = <Proxy user name>
+config.servicemanager.http_proxy_password = <Proxy user password>
 ....
 
-For Windows CMD or Powershell:
+Once these settings are applied, they are passed through to the Docker and OpenShift services to enable them to function correctly.
+In an unauthenticated proxy environment, the `http_proxy_user` and `http_proxy_password` configurations can be omitted.
```
Comment (LalatenduMohanty) [inline docs/installing.adoc]: @Preeticp I am not able to understand purpose of `In an unauthenticated proxy environment, the`http_proxy_user`and`http_proxy_password`configurations can be omitted.` Can you please add more context.

Comment (brgnepal) [reply docs/installing.adoc]: @LalatenduMohanty , this refers to username/password less proxy server url.

**Comment by LalatenduMohanty** [comment]
@Preeticp Looks like there is a bug in VSM  and these settings does not work .Lets wait for PR https://github.com/projectatomic/vagrant-service-manager/pull/435 to merge and then we can rework and merge this PR.

**Comment by LalatenduMohanty** [comment]
@Preeticp https://github.com/projectatomic/vagrant-service-manager/pull/435 is fixed now. Can you please update the PR as per the new code?

File: docs/installing.adoc [hunk 4]
```diff
@@ -277,6 +278,10 @@ host-based tools.
 
 === Manually Download the Vagrant Box
 
+=======
+* Manually Downloading the Vagrant Box Image
++
+>>>>>>> b60c866... Fix #566 Updated Proxy section with the relevant steps from VSM.
```
Comment (LalatenduMohanty) [inline docs/installing.adoc]: @Preeticp Looks like the rebase did not work properly for you. Please clean up the PR.

File: docs/installing.adoc [hunk 5]
```diff
@@ -312,27 +317,54 @@ export BOX=<box-name>
...
-ADB can be set up behind a proxy server. You need to export the proxy
-server information in to the environment and then run `vagrant up`.
-
-NOTE: Currently, only HTTP and HTTPS proxy servers are supported.
-
-For Linux, OS X and Windows Cygwin shell:
+ADB can be set up behind a proxy server. Set the following proxy parameters in the
+Vagrantfile you chose in the previous step:
 
+<<<<<<< HEAD
```
Comment (LalatenduMohanty) [inline docs/installing.adoc]: Same comment as above.

File: docs/installing.adoc [hunk 6]
```diff
@@ -312,27 +317,54 @@ export BOX=<box-name>
...
 export PROXY_USER="foo"
 export PROXY_PASSWORD="mysecretpass"
 ----
+=======
+....
+config.servicemanager.proxy = <Proxy URL>
+config.servicemanager.proxy_user = <Proxy user name>
+config.servicemanager.proxy_password = <Proxy user password>
+....
+>>>>>>> b60c866... Fix #566 Updated Proxy section with the relevant steps from VSM.
```
Comment (LalatenduMohanty) [inline docs/installing.adoc]: Same review comment as above.

File: docs/installing.adoc [hunk 7]
```diff
@@ -312,27 +317,54 @@ export BOX=<box-name>
...
+In an unauthenticated proxy environment, the `proxy_user` and `proxy_password` configurations can be omitted. 
 
+<<<<<<< HEAD
 ----
 setx PROXY="<proxy_server>:<port>"
 setx PROXY_USER="foo"
 setx PROXY_PASSWORD="mysecretpass"
 ----
+=======
+>>>>>>> b60c866... Fix #566 Updated Proxy section with the relevant steps from VSM.
```
Comment (LalatenduMohanty) [inline docs/installing.adoc]: Same comment as above.

**Review by LalatenduMohanty — CHANGES_REQUESTED** [review]

**Review by LalatenduMohanty — APPROVED** [review]

**Comment by LalatenduMohanty** [comment]
LGTM

**Pull request merged**